lib/index.js

```diff
@@ -170,12 +170,6 @@ Attribution.prototype.send = function(path, msg, fn) {
 
   var projectId = this.options.project || window.Attribution.projectId;
 
-  // If we're POSTing, let's send the project_id in the params
-  // only on POST though, or else we'll get two question marks
-  if(cors)
-    url += "?project_id=" + projectId;
-
-  // If we're JSONPing, let's send it in the encoded params
   msg.project_id = projectId;
```

Comment:
@mgz let's use `projectId` when sending to our endpoint, backend still should read both but I'd like to keep it here `camelCase` so it's unified with the rest of segment spec